@here/harp-mapview/lib/VisibleTileSet.ts

```diff
@@ -84,6 +84,15 @@ export interface VisibleTileSetOptions {
      * Number of levels to go down when searching for fallback tiles.
      */
     quadTreeSearchDistanceDown: number;
+
+    /**
+     * Maximal number of new tiles, that can be added to the scene per frame.
+     * if set to `0`the limit will be ignored and all available tiles be uploaded.
+     * @beta
+     * @internal
+     * @defaults 0
```

Comment:
Sorry if this was not communicated properly but it should be `@defaultValue`. See:
https://api-extractor.com/pages/tsdoc/tag_defaultvalue/